indico/modules/events/abstracts/templates/_abstract.html

```diff
@@ -303,3 +331,35 @@ <h3 class="abstract-title">
         </div>
     </div>
 {% endmacro %}
+
+{% macro render_abstract_comments(abstract) %}
+    {% if abstract.comments %}
+        <div id="abstract-comments" class="i-timeline">
+            {% for comment in abstract.comments %}
+                <div class="i-timeline-item">
+                    <div class="i-timeline-item-label action"></div>
+                    <div class="i-timeline-item-box header-indicator">
+                        <div class="i-box-header flex">
+                            <div class="stretch">
+                                {% trans name=comment.user.name, date=comment.created_dt|format_human_date -%}
+                                    <strong>{{ name }}</strong> commented on {{ date }}
+                                {%- endtrans %}
+                                {% if comment.modified_dt %}
+                                    {{ render_edited_hint(comment.modified_dt, comment.modified_by) }}
+                                {% endif %}
+                            </div>
+                            <div class="abstract-comment-badges">
+                                {% if session.user == comment.user or abstract.event_new.can_manage(session.user) %}
```

Comment:
Add a `comment.can_edit(user)` method instead that performs this check. We need it when editing on the server too so we'd replicate protection logic in two places. Guess how likely it is that someone only modifies one of them in the future ;)